server/src/test/java/org/elasticsearch/search/aggregations/metrics/MinAggregatorTests.java

```diff
@@ -740,41 +743,54 @@ public void testShortcutIsApplicable() {
                 )
             );
         }
-        assertNotNull(
-            MinAggregator.getPointReaderOrNull(
-                mockSearchContext(new MatchAllDocsQuery()),
-                null,
-                mockDateValuesSourceConfig("number", true)
-            )
-        );
         assertNull(
             MinAggregator.getPointReaderOrNull(
                 mockSearchContext(new MatchAllDocsQuery()),
                 mockAggregator(),
-                mockDateValuesSourceConfig("number", true)
+                mockDateValuesSourceConfig("number", true, randomFrom(DateFieldMapper.Resolution.values()))
```

Comment:
I'm a little uncomfortable using a randomization for coverage here.  I'd rather loop over the possible resolution values like we loop over the possible numeric types above.

Reply:
Sure!